File: src/main/java/org/theponies/ponecrafter/controller/MenuController.java
package org.theponies.ponecrafter.controller;

import javafx.event.ActionEvent;
import org.theponies.ponecrafter.SceneName;

public class MenuController extends BaseController {

    @Override
    public void setup () {
        init("Menu");
    }

    public void onStartButton (ActionEvent actionEvent) {
        switchScene(SceneName.EDIT_FLOOR);
    }

    public void onExitButton (ActionEvent actionEvent) {
        getStage().close();
    }


}

File: src/main/java/org/theponies/ponecrafter/controller/MenuController.kt
package org.theponies.ponecrafter.controller

import javafx.event.ActionEvent
import org.theponies.ponecrafter.SceneName

class MenuController : BaseController() {

    override fun setup() {
        init("Menu")
    }

    fun onStartButton(actionEvent: ActionEvent) {
        switchScene(SceneName.EDIT_FLOOR)
    }

    fun onExitButton(actionEvent: ActionEvent) {
        stage!!.close()
    }


}
